src/Makefile

```diff
@@ -119,10 +121,26 @@ SIMU_OPTION += -DLAPLACIAN_4TH
 # (b-2-2) WAVE_GRAMFE options
 # ------------------------------------------------------------------------------------
 else ifeq "$(filter -DWAVE_SCHEME=WAVE_GRAMFE, $(SIMU_OPTION))" "-DWAVE_SCHEME=WAVE_GRAMFE"
-# enable for GPU support of Gram-Fourier extension algorithm in GPU mode (CPU is used by default because it is faster)
-# --> must turn on GPU
-#SIMU_OPTION += -DGRAMFE_ENABLE_GPU
+
+# choose gramfe scheme GRAMFE_FFT (uses fast fourier transform, faster for larger patch sizes), GRAMFE_MATMUL (uses matrix multiplication, faster for smaller patches)
+# --> must turn on (SUPPORT_FFTW=FFTW2 and FLOAT8) or (SUPPORT_FFTW=FFTW3) or GPU for GRAMFE_SCHEME == GRAMFE_FFT
+SIMU_OPTION += -DGRAMFE_SCHEME=GRAMFE_FFT
```

Comment:
I think we should use `GRAMFE_MATMUL` by default, but its mass conservation properties are definitely still worse than that of the FFT version. However, since the fixup flux operation and the interpolation cause much bigger errors, I think it is okay to use it since it is much faster. I will write that in the manual.